<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherWindow</class>
 <widget class="QMainWindow" name="WeatherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWeight:{
	border-radius: 20px;     /* 边框圆角 */
}

QLineEdit{
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	padding-left: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;     /* 边框圆角 */
background-color: rgb(211, 211, 211);</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>751</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 183, 119);
background-color: rgb(145, 145, 145);
border-radius: 20px;     /* 边框圆角 */</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;     /* 边框圆角 */</string>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;     /* 边框圆角 */</string>
      </property>
      <property name="placeholderText">
       <string>城市</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#search_button{
	image: url(:/png/search_normal.png);
	border-radius:15px;
}
#search_button:hover{
	image: url(:/png/search_hover.png);
	border-radius:15px;
}
#search_button:pressed{
	image: url(:/png/search_press.png);
	border-radius:15px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="wea_icon">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>×</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="today_weight" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>751</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 239, 243);</string>
     </property>
     <property name="text">
      <string>   </string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>天气图标</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 233, 243);</string>
     </property>
     <property name="text">
      <string>    </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 240, 243);</string>
     </property>
     <property name="text">
      <string>   </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>701</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 237, 246);
border-radius: 10px; </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>70</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 228, 232);</string>
     </property>
     <property name="text">
      <string>     </string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>701</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 181, 181);
background-color: rgb(218, 193, 202);</string>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>651</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_11"/>
      <widget class="QLineEdit" name="lineEdit_12"/>
      <widget class="QLineEdit" name="lineEdit_13"/>
      <widget class="QLineEdit" name="lineEdit_14"/>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="text">
        <string>空气质量</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="text">
        <string>pm2.5</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="text">
        <string>湿度</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="text">
        <string>风速</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>161</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_25">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 202, 210);</string>
     </property>
     <property name="text">
      <string>  </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="futrue_weight" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>751</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 251, 252);</string>
    </property>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_21">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_22">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_23">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_24">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
